dragos-worldview-threat-intelligence/operations.py

```diff
@@ -96,12 +96,36 @@ def build_query(params, query_params):
         raise ConnectorError(str(e))
 
 
-def get_all_indicators(config, params):
+def get_indicators_response(config, params):
     query_params = build_query(params, indicator_params)
     endpoint = 'indicators' + (('?' + query_params) if query_params else '')
     return make_rest_call(config, endpoint)
 
 
+def get_all_indicators(config, params):
+    fetch_all_records = False
+    if not params.get("page") and not params.get("page_size"):
+        fetch_all_records = True
+    if fetch_all_records:
+        page = 1
+        indicators = list()
+        params.update({"page": page, "page_size": 1000})
+        response = get_indicators_response(config, params)
+        resp_indicators = response.get("indicators", [])
```

Comment:
I feel, this code has potential to optimise more.

Reply:
@cs-suyog-jinturkar this code was tested for 3739 indicators. Can you please suggest how to optimise further ?

Reply:
@cs-suyog-jinturkar I have done the suggested changes